### ISSUE TITLE
`check-added-large-files`: InvalidManifestError

### ISSUE THREAD

**Issue by Rohlik** [pre-commit/pre-commit-hooks#1109]
Hello 👋🏼,
I have a issue with the `check-added-large-files` hook, see the error below:
```
$ pre-commit run --all-files
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
An error has occurred: InvalidManifestError: 
==> File /home/notebook/.cache/pre-commit/repo51f9mfvh/.pre-commit-hooks.yaml
==> At Hook(id='check-added-large-files')
==> At key: stages
==> At index 0
=====> Expected one of commit, commit-msg, manual, merge-commit, post-checkout, post-commit, post-merge, post-rewrite, prepare-commit-msg, push but got: 'pre-commit'
Check the log at /home/tro/.cache/pre-commit/pre-commit.log
```

That mentioned YAML file contains this related part:
```yaml
-   id: check-added-large-files
    name: check for added large files
    description: prevents giant files from being committed.
    entry: check-added-large-files
    language: python
    stages: [pre-commit, pre-push, manual]
    minimum_pre_commit_version: 3.2.0
```

Can you help me fix it?

**Comment by asottile** [comment]
try using your favorite search engine for the error message or reading the release notes

**Comment by Rohlik** [comment]
I had to uninstall that package from my system as it can't be updated to a newer version - I was using elementary Ubuntu package repositories.
So I installed it via `pip` and it does not work.

**Comment by waipeng** [comment]
Hope this helps someone like me who stumble upon this thread.

Commit 003dfa55c9eb05b08b33f60864363392958f4618 introduced in pre-commit-hooks v5.0.0 is incompatible with older pre-commit version. I am on Ubuntu 22.04, which ships with pre-commit 2.17.0-1.

There have been an attempt to fix it in [047439abffb164edd5b49e50439fd63a625be3da](https://github.com/pre-commit/pre-commit/commit/047439abffb164edd5b49e50439fd63a625be3da) in pre-commit v3.6.0, but that is still a newer version of what is available on Ubuntu 22.04 (out of the box).

TL;DR; either
1. Upgrade pre-commit to v3.2.0
2. Use pre-commit-hooks v4.6.0

Hope that helps!

**Comment by dhimmel** [comment]
Noting duplicate issues to help with lookup (I arrived at the dups first and there was no link back to this issue.

- https://github.com/pre-commit/pre-commit-hooks/issues/1125
- https://github.com/pre-commit/pre-commit-hooks/issues/1128
